_timeline/2018-09-29-mesh-picnic.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Mesh Picnic
+date: 2018-09-29
+---
+[Free Geek Toronto](https://www.freegeektoronto.org/) hosted a technology picnic open to the public.  
```

Comment:
The double-space at the end of this line intentional? The line-break seems off to me:
![screen shot 2018-10-09 at 9 48 10 pm](https://user-images.githubusercontent.com/227587/46708523-0aa49700-cc0d-11e8-83b3-838d6dea65e4.png)

Reply:
Timeline should be one blob of text like the rest, this looks off.

Reply:
Thanks @garrying @benhylau - it's one blob of text now.